library/Backends/Michelson/src/Juvix/Backends/Michelson/DSL/InstructionsEff.hs

```diff
@@ -813,7 +813,7 @@ apply closure args remainingArgs = do
       throw @"compilationError" (Types.InternalFault "apply to non lam")
 
 -- we can only delete things at position greater than 0.
--- this is because if we were to delete 0, then (λx : ω i. x) would error
+-- this is because if we were to delete 0, then (λx : _ i. x) would error
```

Comment:
Probably shuold find a new symbol for this, as it's not clear with this what would error